controller_manager/include/controller_manager/controller_manager.h

```diff
@@ -167,6 +167,11 @@ class ControllerManager{
 private:
   void getControllerNames(std::vector<std::string> &v);
 
+  void manageSwitch(const ros::Time &time);
```

Comment:
We use "typename& arg_name" in this codebase for references.

Reply:
Fixed